content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,117 @@
 ---
 title: "WebRTC"
-description: "WebRTC is a new specification that uses QUIC to offer an alternative to WebSocket. Conceptually, it can be considered WebSocket over QUIC.Learn about WebTransport and how it is used in libp2p."
+description: "WebRTC is a protocol for real-time communication and is used to establish connections between browsers and other nodes."
 weight: 110
 aliases:
     - "/concepts/transports/webrtc"
 ---
 
-Coming soon!
+## What is WebRTC?
```

Comment:
We should add a sentence or two above this section what WebRTC is used for in libp2p.

Reply:
I added something similar to "WebRTC in libp2p" as it is better suited with this format.